src/runtime/contrib/coreml/coreml_runtime.mm

```diff
@@ -174,17 +158,17 @@
         CHECK(args[i].type_code() == kTVMDLTensorHandle || args[i].type_code() == kTVMNDArrayHandle)
             << "Expect NDArray or DLTensor as inputs\n";
         if (args[i].type_code() == kTVMDLTensorHandle) {
-          model.SetInput([input_names[i] UTF8String], args[i]);
+          model_->SetInput([input_names[i] UTF8String], args[i]);
         } else {
           LOG(FATAL) << "Not implemented";
         }
       }
 
       // Execute the subgraph.
-      model.Invoke();
+      model_->Invoke();
 
       // TODO: Support multiple outputs.
-      NDArray out = model.GetOutput(0);
+      NDArray out = model_->GetOutput(0);
       if (args[args.size() - 1].type_code() == kTVMDLTensorHandle) {
         DLTensor* arg = args[args.size() - 1];
         out.CopyTo(arg);
```

Comment:
Are the ndarrays allocated by the host for both inputs and outputs? If so, I think we might be able to do zero copy (in a separate PR).

Reply:
Yes, will do it in the next PR :)